src/workerd/tools/BUILD.bazel

```diff
@@ -13,13 +13,42 @@ wd_cc_binary(
     ],
 )
 
-run_binary(
+compat_dates = [
+    # Oldest compatibility date, with no flags enabled
+    ("2021-01-01", "oldest"),
+    # https://developers.cloudflare.com/workers/platform/compatibility-dates/#formdata-parsing-supports-file
+    ("2021-11-03", "2021-11-03"),
+    # https://developers.cloudflare.com/workers/platform/compatibility-dates/#settersgetters-on-api-object-prototypes
+    ("2022-01-31", "2022-01-31"),
+    # https://developers.cloudflare.com/workers/platform/compatibility-dates/#global-navigator
+    ("2022-03-21", "2022-03-21"),
+    # https://developers.cloudflare.com/workers/platform/compatibility-dates/#global-navigator
+    ("2022-08-04", "2022-08-04"),
+    # Latest compatibility date
+    ("2024-01-01", "experimental"),
```

Comment:
I've pushed it to `2030`. I don't think we want to enable things without a default on date, because that includes _super_ experimental stuff like nodejs compat

Reply:
Maybe experimental is the wrong name then, actually—I had `latest` before, but that didn't feel sufficiently warning-y

Reply:
What's the purpose of this / how does this string get exposed to the customer? My (fuzzy) understanding is that this code will generate type information from enabling all compatibility flags which have enable-dates, and this type information is named "experimental"?

"latest" would make more sense to me, if so. When we assign an enable-date to a compatibility flag, we are saying that the new behavior is now the preferred, stable behavior.

Reply:
The issue that we saw with `latest` was that if someone is using that set of types in their typescript definitions, an update to the package would unexpectedly change their types (because new compatibility flags will be added and integrated into `latest`/`experimental` in the future), whereas pinning to a date version will guarantee type consistency as of that default on date. Currently, we think this'll be exposed as a reference to `@cloudflare/workers-types/experimental` or `@cloudflare/workers-types/2022-11-03` in a user's typescript config.

Reply:
I think what Harris is saying is call it `latest` or `now` instead of `experimental`. `experimental` is misleading as to the name of the types. I concur with that.

What's going to confuse me as an end reader is that `@cloudflare/workers-types/2022-11-03` is totally divorced from the compat date in `wrangler.toml`. In `wrangler.toml` I can pick *any* date. In the types I can only pick dates that mean something in the runtime. This is going to be a point of friction and confusion. You may want to make sure the docs are very clear here about guidance (e.g. "make sure that wrangler.toml picks the same date as the types for readability"). Alternatively, if you can provide types for all the dates such that they symlink to the real dates that have type signatures generated, then you can just say "specify the date same as you do in wrangler.toml" without forcing them to reference the (lagging & incomplete) docs to find the date they want.